decide/src/main/java/group17/LicAnalyzer.java

```diff
@@ -153,7 +153,52 @@ public boolean lic8(InputHandler input) {
         return false;
     }
 
-    public boolean lic9(InputHandler input) {
+    /**
+     *  Checks if a set of three points is separated by C_PTS and D_PTS, and if angle condition is met.
+     *  @returns true, false, or @throws IllegalArgumentException
+     *  @param input.NUMPOINTS,C_PTS,D_PTS,EPSILON,X_COORD,Y_COORD
+    */
+    public boolean lic9(InputHandler input) throws IllegalArgumentException {
+
+        if(input.NUMPOINTS < 5 || input.NUMPOINTS > 100)
+            throw new IllegalArgumentException("Exception thrown from: LIC 9. Reason: NUMPOINTS < 5 or NUMPOINTS > 100.");
+        else if(input.C_PTS < 1 || input.D_PTS < 1 )
+            throw new IllegalArgumentException("Exception thrown from: LIC 9. Reason: C_PTS < 1 or D_PTS < 1.");
+        else if(input.C_PTS + input.D_PTS > input.NUMPOINTS - 3)
+            throw new IllegalArgumentException("Exception thrown from: LIC 9. Reason: C_PTS + D_PTS > NUMPOINTS - 3");
+        GeometryUtils geom = new GeometryUtils();
```

Comment:
Minor but I think there is a `GeometryUtils` instance instantiated at the beginning of the `LicAnalyzer` class that you could use (maybe it was added in a later merge then ignore this).